packages/flutter_tools/lib/src/debug_adapters/flutter_adapter.dart

```diff
@@ -192,15 +192,11 @@ class FlutterDebugAdapter extends DartDebugAdapter<FlutterLaunchRequestArguments
 
   @override
   Future<void> debuggerConnected(vm.VM vmInfo) async {
-    // Capture the PID from the VM Service so that we can terminate it when
-    // cleaning up. Terminating the process might not be enough as it could be
-    // just a shell script (e.g. flutter.bat on Windows) and may not pass the
-    // signal on correctly.
-    // See: https://github.com/Dart-Code/Dart-Code/issues/907
-    final int? pid = vmInfo.pid;
-    if (pid != null) {
-      pidsToTerminate.add(pid);
-    }
+    // Usually we'd capture the pid from the VM here and record it for
+    // terminating, however for Flutter apps it may be running on a remove
```

Comment:
```suggestion
    // terminating, however for Flutter apps it may be running on a remote
```

Reply:
Thanks! I guess it's a good time to test that CLA Bot :-)

Reply:
Neat :-)

---

The following contributors were found for this pull request:

✅ https://github.com/flutter/flutter/commit/759fc936679bff376e5ce4ee15626982a691a60a Author: @DanTup <d***y​@tuppeny.com>
✅ https://github.com/flutter/flutter/commit/759fc936679bff376e5ce4ee15626982a691a60a Co-Author: <fu***o​@google.com>

(Only the first commit for a unique contributor is listed.)